holidays/calendars/hindu.py

```
@@ -2072,6 +2829,84 @@ class _HinduLunisolar:
+        2025: (AUG, 8),
+        2026: (AUG, 28),
+        2027: (AUG, 13),
+        2028: (AUG, 4),
+        2029: (AUG, 24),
+        2030: (AUG, 9),
+        2031: (AUG, 1),
+        2032: (AUG, 20),
+        2033: (AUG, 5),
+        2034: (AUG, 25),
+        2035: (AUG, 17),
+    }
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)